projects/common/src/index.ts

```diff
@@ -396,26 +420,27 @@ export const frontPath = (issue: string, frontId: string) =>
 
 // These have issueids in the path, but you'll need to change the archiver if you want to use them.
 
+//To remove.
```

Comment:
when?

Reply:
never, :(
I was planning on moving `media` to `images/${size}/full-size`
but instead I just moved all the others to `thumbs/${size}/${use}`
this means that the image paths don't change for full size images